editor.html

```diff
@@ -227,10 +227,23 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                     tabindex="7"
                     title="Create a link to share your script">
                     <div class="roundsymbol">
-                        <i class="fa fa-share-alt"></i>
+                      <i class="fa fa-share-alt"></i>
```

Comment:
I might have introduced this whitespace change, sorry!
Could you add the extra spaces to reduce the final diff?

Reply:
Done